tidb-in-kubernetes/k8s-deployment.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 部署 k8s
+category: how-to
+---
+
```

Comment:
- Use only one first level heading.
- Add a blank line both before and after the title.

```suggestion
# Kubernetes 上的 TiDB 集群环境需求
```